ww3tools/modelSat_collocation.py

```diff
@@ -134,6 +137,12 @@
 if len(sys.argv) > 7:
 	sys.exit(' Too many inputs')
 
+# Quality control parameters
+qc="no"
```

Comment:
@ricampos I thought you wanted to have this default to yes?

Reply:
Thanks for noticing. 
I'm still modifying modelSat_collocation using the configuration arguments from ww3tools.yaml.
Next version is coming soon and it will be the same as in ProcSat_Altimeter.py